PayFineUI.java

```diff
@@ -13,7 +13,7 @@ public PayFineUI(PayFineControl control) {
 		this.control = control;
 		keyboardinput = new Scanner(System.in);
 		fineUiState = PayFineUiState.INITIALISED;
-		control.payFineUi(this);
+		control.setPayFineUi(this);
 	}
 
 	public void setFIneUiState(PayFineUiState state) {

```

Comment:
setFIneUiState is not to the code standards  as there is a capital i in FIne should be setFineUiState

Reply:
Okay